packages/safe-core-sdk/src/Safe.ts

```diff
@@ -529,6 +529,14 @@ class Safe {
       )
     }
 
+    const value = BigNumber.from(safeTransaction.data.value)
+    if (value.gt(BigNumber.from(0))) {
```

Comment:
```suggestion
    if (!value.isZero()) {
```
according to https://docs.ethers.io/v5/api/utils/bignumber/ such method is available